### PR TITLE
Add servers amd images unittest

### PR DESCRIPTION
**Pull request by ajiang38740**

### CODE REVIEW

File: tests/jumpgate-tests/image/test_images.py [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+        retExpected = self.client['Virtual_Guest_Block_Device_Template_Group']
+        retExpected.getPrivateBlockDeviceTemplateGroups.return_value = []
+        ret = self.instance.get_private_images(self.guid, self.name,
+                                               self.limit, self.marker)
+        queryMock.assert_any_call(self.name)
+        queryMock.assert_any_call(self.guid)
+        queryMock.assert_any_call('> %s' % self.marker)
+        self.assertEquals(type(ret), list)
+
+    def tearDown(self):
```
Comment (sudorandom) [inline tests/jumpgate-tests/image/test_images.py:161]: This kind of teardown isn't needed... A new instance of the class is stood up between each test run.

**Comment by bhaktadn** [comment]
Since Alan is out, it will be difficult to handle the requested changes in his branch. I think the easiest way to handle the changes for the comment is merge this request as is and then I create a new pull request to handle the comment. Is it possible?

**Comment by sudorandom** [comment]
@bhaktadn You can pull in the commits associated with this pull request to a branch on your own fork and submit a new pull request with additional changes. 

Here's an example of pulling in his changes:

```
git checkout -b ajiang38740-add_servers_unittest master
git pull https://github.com/ajiang38740/jumpgate.git add_servers_unittest
```

**Comment by bhaktadn** [comment]
Thanks. I have created a new pull request 133. Please close this one.

**Pull request merged**